Tables/FE8 Nightmare modules/Quote editors/Death Quote Editor.csv

```diff
@@ -1,4 +1,4 @@
-INLINE NewDeathQuoteTable,Character,##UNKNOWN##,Chapter,Event ID,##UNKNOWN##,Text ID,##UNKNOWN##
+0x9ECD4C,Character,##UNKNOWN##,Chapter,Event ID,##UNKNOWN##,Text ID,##UNKNOWN##
```

Comment:
:(

Reply:
I second that :(.
Why you do dis?

Reply:
Please revert this change and send me the `Death Quote Editor.event` that's generated during the MAKE HACK process.